scripts/pre_push_hook.py

```diff
@@ -124,10 +124,13 @@ def _get_remote_name():
         raise ValueError(err)
 
     if not remote_num:
-        print ('Warning: Please set upstream for the lint checks to run '
-               'efficiently. You can learn more about it here -> '
-               'https://git-scm.com/book/en/v2/Git-Branching-Remote-Branches\n')
-        return
+        raise Exception(
+            'Error: Please set upstream for the lint checks to run '
```

Comment:
Can we make this message more explicit, and just tell the developer exactly what to do? They shouldn't have to read a long web page in order to figure out what's wrong.

Reply:
Done.